tempesta_fw/sched/tfw_sched_http.c

```diff
@@ -302,6 +304,13 @@ tfw_sched_http_cfg_handle_match(TfwCfgSpec *cs, TfwCfgEntry *e)
 	memcpy(rule->rule.arg.str, in_arg, arg_len);
 	rule->rule.arg.type = tfw_sched_http_cfg_arg_tbl[field];
 
+	if (rule->rule.field == TFW_HTTP_MATCH_F_HDR_RAW) {
+		p = rule->rule.arg.str;
+		while ((*p = tolower(*p))) {
+			p++;
+		}
+	}
```

Comment:
This is http_match specific code and you already has 2 instances of the code, so it's good to move as function to http_match.[ch]